.eslintrc.js

```diff
@@ -31,7 +31,11 @@ const tsRuleOverrides = {
   '@typescript-eslint/no-for-in-array': 'off',
 
   // Optional chaining compiles to a lot more code
-  '@typescript-eslint/prefer-optional-chain': 'off'
+  '@typescript-eslint/prefer-optional-chain': 'off',
+
+  // Support TypeScript 3.8 by disallowing import { type Module } from 'module'
+  // TODO: Find the export equivalent
```

Comment:
how about https://typescript-eslint.io/rules/consistent-type-exports/ ?

Reply:
I did try this one! It gives the same behaviour as we have now, where it will allow `export { type ReadonlySampler, type ReadWriteSampler }` if you type it manually